instrumented/integration/src/main/kotlin/com/datadog/android/sdk/integration/RuntimeConfig.kt

```diff
@@ -65,4 +70,5 @@ internal object RuntimeConfig {
 
     const val DD_TOKEN = "MYTESTAPPTOKEN"
     const val DD_CONTENT_TYPE = "application/json"
+    const val TRACES_SERVICE_NAME = "android"
```

Comment:
Isn't that the default name of the tracer ?

Reply:
Yup, you're right I don't need this here.